modules/videoio/cmake/detect_obsensor.cmake

```diff
@@ -1,28 +1,53 @@
 # --- obsensor ---
 if(NOT HAVE_OBSENSOR)
```

Comment:
We should distribute licenses of used 3rdparty code (not only for downloaded cases but also if we use preinstalled SDKs).
Please use `ocv_install_3rdparty_licenses` macro with appropriate file from SDK.

Reply:
> We should distribute licenses of used 3rdparty code (not only for downloaded cases but also if we use preinstalled SDKs). Please use `ocv_install_3rdparty_licenses` macro with appropriate file from SDK.

I have added relevant operations.